docs/DOCS_GUIDELINES.md

```diff
@@ -22,19 +24,19 @@ Relative links should be used nearly everywhere, due to versioning. Note that in
 
 Code snippets can be included in the documentation using normal Markdown code blocks. For example:
 
-```md
+````md
```

Comment:
4 backticks?

Reply:
Yes, markdown also accepts this. My linter did this because it helps differentiate nested code blocks.